[kde-plasma/plasma-desktop] Split up USE=usb into evdev,logitech

a17r opened the pull request:
Makes virtual/libusb:0 optional through USE=logitech
Keeps kcm_input optional through USE=evdev (and re-adjusts DEPENDs)

Package-Manager: portage-2.2.20

johu commented:
Please rebase latest changes. I have already removed the patch in kde overlay master.

a17r commented:
Yes, certainly makes a lot of sense to be enabled by default. Removed evdev USE flag again to avoid confusion, it can be added any time should there be some libinput alternative or use of plasma without mouse in the future. Also moved to DEPEND only as just a header is required.

Also dropped the disabling of packagekitqt5 - thought it could prevent trouble by the time such a package goes into portage.

a17r commented:
Another day another situation - upstream made libusb:0 dependency optional (it's for very old Logitech devices in fact) and now evdev flag does what was previously enabled by usb.

johu commented:
Lets wait for the review then.

johu commented:
Thanks. This is rebased onto kde overlay master. Quizzzzzzzzzzzzzzzzzz! :)
